CHANGELOG.asciidoc

```diff
@@ -3,6 +3,108 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.3.0]]
+=== Beats version 8.3.0
+https://github.com/elastic/beats/compare/v8.2.3\...v8.3.0[View commits]
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+- Allow loading secrets that contain commas from the keystore {pull}31694{pull}.
```

Comment:
```suggestion
- Allow loading secrets that contain commas from the keystore {pull}31694[31694].
```
The pull request link for this item doesn't work properly in the 8.3.0 release notes.

Reply:
Thanks, will be fixed by: https://github.com/elastic/beats/pull/32156/commits/e507fe43a97753247f4bc28fa0565a725e7285ed